scanner/cmd/updater/main.go

```diff
@@ -19,8 +20,31 @@ func main() {
 		log.Fatal("Missing argument for the output directory.")
 	}
 
-	ctx := context.Background()
-	if err := updater.Export(ctx, *outputDir); err != nil {
-		zlog.Error(ctx).Err(err).Send()
+	const maxRetries = 3
+	const timeoutDuration = 35 * time.Minute
+	var err error
+
+	for attempt := 1; attempt <= maxRetries; attempt++ {
+		ctx, cancel := context.WithTimeout(context.Background(), timeoutDuration)
+		defer cancel()
+
+		err = updater.Export(ctx, *outputDir)
+		if err == nil {
+			break
+		}
+
+		if ctx.Err() == context.DeadlineExceeded {
```

Comment:
```suggestion
		if errors.Is(ctx.Err(), context.DeadlineExceeded) {
```

You would need to import `"errors"` (note: NOT `"github.com/pkg/errors"`)